SWIFTLINT.md

```diff
@@ -0,0 +1,138 @@
+# Swift Style Guide
+
+The SwiftLint configuration on this repo is based on the great work of the raywenderlich.com folks [The Official raywenderlich.com Swift Style Guide](https://github.com/raywenderlich/swift-style-guide).
+
+The purpose of this guide is to provide a base format for our Swift codebase. This
+guide describes what are the rules enforced by our swift lint rules and should be 
+used when contributing or reviewing code. 
+
+These guides use SwiftLint as a standard. You can learn more about SwiftLint by visiting its [official documentation page](https://github.com/realm/SwiftLint).
+
+## Table of Contents
+
+* [Installing SwiftLint](#installing-swiftlint)
+* [Using the configuration file](#using-the-configuration-file)
+* [Xcode settings](#xcode-settings)
+* [Running SwiftLint](#running-swiftlint)
+* [Handling rule exceptions](#handling-rule-exceptions)
+* [Approved exceptions](#approved-exceptions)
+	* [Implicitly Unwrapped Optionals](#implicitly-unwrapped-optionals)
+	* [Force cast](#force-cast)
+	* [Force unwrapping](#force-unwrapping)
+	* [SwiftUI and multiple trailing closures](#swiftui-and-multiple-trailing-closures)
+	* [Open-source files](#open-source-files)
+* [Other notes](#other-notes)
+
+## Installing SwiftLint
+
+
+```bash
```

Comment:
I never knew of this syntax highlighting feature until just now 🤯.

Very minor but does GitHub support `zsh`?  Although it's basically the same as `bash` for most usages, macOS switched to zsh as the default as of Catalina.

Reply:
I looked up the GitHub syntax highlighting [documentation](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml), and it all resolves to a "shell script" highlighter.  So either "bash" or "zsh" would have the same result here.